package.json

```diff
@@ -34,14 +34,17 @@
     "postversion": "git push && git push --tags"
   },
   "dependencies": {
-    "@aws-sdk/client-codeartifact": "^1.0.0-rc.7",
+    "@aws-sdk/client-codeartifact": "^3.0.0",
+    "@aws-sdk/credential-provider-node": "^3.0.0",
     "@types/node": "^14.0.0",
     "@yarnpkg/builder": "^2.1.3",
     "@yarnpkg/core": "^2.4.0",
     "@yarnpkg/fslib": "^2.4.0",
     "@yarnpkg/parsers": "^2.3.0",
     "@yarnpkg/plugin-npm": "^2.4.0",
     "clipanion": "^2.6.2",
+    "lodash.defaultsdeep": "^4.6.1",
+    "lodash.get": "^4.4.2",
     "typescript": "^3.9.5",
```

Comment:
the latest typescript version is already at 4.1.3
maybe this would be a good time to upgrade the dependencies and bring them all up to latest?